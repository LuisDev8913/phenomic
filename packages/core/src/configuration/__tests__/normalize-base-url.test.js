import normalizeBaseUrl from "../normalize-base-url.js";

it("should normalize base url with a trailing slash", () => {
  expect(normalizeBaseUrl("http://t.e/st")).toMatchSnapshot();
});
