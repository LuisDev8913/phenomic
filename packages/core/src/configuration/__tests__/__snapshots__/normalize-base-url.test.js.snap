// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should normalize base url with a trailing slash 1`] = `
Object {
  "auth": null,
  "hash": null,
  "host": "t.e",
  "hostname": "t.e",
  "href": "http://t.e/st/",
  "path": "/st/",
  "pathname": "/st/",
  "port": null,
  "protocol": "http:",
  "query": null,
  "search": null,
  "slashes": true,
}
`;
